agent/ecscni/plugin.go

```diff
@@ -110,7 +133,21 @@ func (client *cniClient) SetupNS(cfg *Config) (*current.Result, error) {
 
 // CleanupNS will clean up the container namespace, including remove the veth
 // pair and stop the dhclient
-func (client *cniClient) CleanupNS(cfg *Config) error {
+func (client *cniClient) CleanupNS(ctx context.Context, cfg *Config) error {
```

Comment:
Same comment about accepting timeout as a param and dealing with `ctx.Err()` as above.